infrastructure/modules/products/app-registrations.tf

```diff
@@ -1,8 +1,7 @@
 resource "azuread_application" "pars-upload" {
   name                    = "pars-upload-${var.environment}"
-  reply_urls              = concat(["https://${azurerm_cdn_endpoint.pars.host_name}"], var.add_local_pars_reply_url ? ["http://localhost:3000"] : [])
+  reply_urls              = concat(["https://${azurerm_cdn_endpoint.pars.host_name}", "https://${azurerm_storage_account.pars.primary_web_host}", "https://mhraparsnonprod.z33.web.core.windows.net", "https://mhraparsnon-prod.azureedge.net"], var.add_local_pars_reply_url ? ["http://localhost:3000"] : [])
   group_membership_claims = "SecurityGroup"
```

Comment:
It would be better to pass in hardcoded, environment-specific values as var's.

Reply:
I'm starting to think we just hardcode (or environment var) the lot and have one app registration

Reply:
@majikandy if we do that, could we include it in each environment's `main.tf`, so no matter which environment we're terraforming, it checks that the app registration is there?

Reply:
Agreed with @TimboTambo we will keep the separate app registrations just incase we want to try something with the app registration and no affect prod app registration.

So probably the happy medium is that we have one for all non prod and one for prod.